spec/system/school/users/users_show_spec.rb

```diff
@@ -57,6 +65,7 @@
     expect(page).to have_text(user.email)
     expect(page).to have_text(DiscordRole.first.name)
     expect(page).to have_text("##{user.discord_user_id}")
+    expect(page).to have_text(user.affiliation)
```

Comment:
This check is not required here as in L69, organisation name presence is already being checked